libc-bottom-half/headers/public/wasi/api.h

```diff
@@ -1277,15 +1263,12 @@ typedef union __wasi_subscription_u_u_t {
     __wasi_subscription_fd_readwrite_t fd_write;
 } __wasi_subscription_u_u_t;
 typedef struct __wasi_subscription_u_t {
-    __wasi_eventtype_t tag;
+    uint8_t tag;
```

Comment:
this got less precise, even though __wasi_eventtype_t is typedef'd to uint8_t.

Reply:
That's true, yeah, and is due to tags no longer being modeled as pointers to an `enum` but rather just simply having a particular bit-width as desired. This means that the documentation is lost a bit but at the bindings layer everything should be the same since `__wasi_eventtype_t` is a typedef to `uint8_t` anyway.

Reply:
sgtm